Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2004-08-17
Initial Package Version: 3.0
Origin: Paul Jarc
Upstream Status: Sent
Description: Fixes a display issue which will only show 33 characters
             then wrap to the next line.
 
--- bash-3.0/lib/readline/display.c.orig	2004-08-17 20:24:39.290067304 +0000
+++ bash-3.0/lib/readline/display.c	2004-08-17 20:24:44.432285568 +0000
@@ -351,14 +351,14 @@
       local_prompt = expand_prompt (p, &prompt_visible_length,
 				       &prompt_last_invisible,
 				       (int *)NULL,
-				       (int *)NULL);
+				       &prompt_physical_chars);
       c = *t; *t = '\0';
       /* The portion of the prompt string up to and including the
 	 final newline is now null-terminated. */
       local_prompt_prefix = expand_prompt (prompt, &prompt_prefix_length,
 						   (int *)NULL,
 						   &prompt_invis_chars_first_line,
-						   &prompt_physical_chars);
+						   (int *)NULL);
       *t = c;
       return (prompt_prefix_length);
     }
